controllers/host/host_controller.go

```diff
@@ -2073,6 +2147,78 @@ func (r *HostReconciler) Reconcile(ctx context.Context, request ctrl.Request) (r
 		return common.RetrySystemNotReady, nil
 	}
 
+	if !r.CloudManager.GetHostUpdateRoutinesRunning() {
+		logHost.Info("Starting goroutines to receive host strategy updates from other controllers")
+		go func() {
+			for {
+				host_strategy := r.CloudManager.ReceiveHostStrategyUpdate()
+				host_instance := &starlingxv1.Host{}
+				host_namespace := types.NamespacedName{Namespace: request.NamespacedName.Namespace, Name: host_strategy.Host.Hostname}
+				err = r.Client.Get(context.TODO(), host_namespace, host_instance)
+				if err != nil {
+					logHost.Error(err, "Failed to get resource for host namespace %v", host_namespace)
+					continue
+				}
+
+				if host_instance.Status.StrategyRequired != host_strategy.StrategyRequired {
+					host_instance.Status.StrategyRequired = host_strategy.StrategyRequired
+					err := r.Client.Status().Update(context.TODO(), host_instance)
+					if err != nil {
+						logHost.Error(err, "Failed to update status %v", host_instance.Status)
+						continue
+					}
+				}
+
+				if !r.CloudManager.GetStrageySent() {
+					msg := fmt.Sprintf("Applying %s strategy on host %s", host_strategy.StrategyRequired, host_instance.Name)
+					logHost.Info(msg)
+					r.CloudManager.SetResourceInfo(cloudManager.ResourceHost, host_strategy.Host.Personality, host_instance.Name, host_instance.Status.Reconciled, host_instance.Status.StrategyRequired)
```

Comment:
create variable to keep this line shorter. At least for host_instance.Status.Reconciled and host_instance.Status.StrategyRequired

Reply:
> create variable to keep this line shorter. At least for host_instance.Status.Reconciled and host_instance.Status.StrategyRequired

Done